st2api/st2api/controllers/v1/packviews.py

```diff
@@ -110,8 +115,10 @@ def get_one(self, name_or_id, *file_path_components):
             # Ignore references to files which don't exist on disk
             raise StackStormDBObjectNotFoundError('File "%s" not found' % (file_path))
 
-        content = self._get_file_content(file_path=normalized_file_path)
-        return content
+        response.headers['Cache-Control'] = 'public, max-age=86400'
```

Comment:
I think directly returning new `webob.Response` object is safer than modifying a global variable. That's also what we do in other places.

Besides that, LGTM, +1.